sections/builder.js

```diff
@@ -60,7 +60,7 @@ export default [
     type: "method",
     method: "as",
     example: ".as(name)",
-    description: "Allows for aliasing a subquery, taking the string you wish to name the current query. If the query is not a sub-query, it will be ignored.",
+    description: "Allows for aliasing a subquery, taking the string you wish to name the current query. NOTE: Does not create SQL alias for columns or tables.",
```

Comment:
I don't understand what you mean by this addition / change.

Reply:
I feel the documentation should be more clear that the as() function only works on sub-queries.

Reply:
fair enough, I don't really know how to word this better. I guess its fine if we just leave it as is.